.github/scripts/label_utils.py

```diff
@@ -24,10 +24,10 @@
 LABEL_ERR_MSG = f"""# {LABEL_ERR_MSG_TITLE}
 If your changes are user facing and intended to be a part of release notes, please use a label starting with `release notes:`.
```

Comment:
Can you change this to: 

If your changes are intended to be a part of release notes, please use a label starting with `release notes:

"User facing" is a bit confusing in general, let's just remove it. The authors will know if it's appropriate to include in the release notes.